### DISCUSSION TITLE
Welcome to nabarr Discussions!

### DISCUSSION THREAD

**Discussion by l3uddz** [l3uddz/nabarr#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by 245123** [comment]
Would be a good idea to add an example of working rss feed. Оn all my feeds, I get a message **Refreshed, no items to queue**.
